packages/maskbook/src/plugins/Polls/UI/Polls.tsx

```diff
@@ -126,9 +135,12 @@ export function PollCardUI(props: PollCardProps) {
                     <ListItem
                         className={classes.option}
                         key={index}
-                        onClick={() => {
-                            vote?.(poll, index)
-                        }}>
+                        onClick={debounce(
```

Comment:
IMO it will create a new "debounced" button every time so it will not work

Reply:
@1999-XiChi cc